internal/jobqueue/job.go

```diff
@@ -52,50 +53,58 @@ func (job *Job) Run() error {
 	}
 	err = json.NewDecoder(stdout).Decode(&result)
 	if err != nil {
-		return err
+		return err, nil
 	}
 
 	err = cmd.Wait()
 	if err != nil {
-		return err
+		return err, nil
 	}
 
+	r := []error(nil)
```

Comment:
`var r []error` is imho easier to write but this is just small nit, feel free to use your approach. :)

Reply:
Yeah, good point.